### DISCUSSION TITLE
Prevent Helix from scrolling too far past the EOF

### DISCUSSION THREAD

**Discussion by dist1ll** [helix-editor/helix#6965] [Q&A] [answered]
Between v22.12 and v23.03, the scrolling behavior at the bottom of a file changed. 

Now when I scroll to the very bottom, instead of showing only ~8 lines past the end of the file, helix allows me to scroll down almost an entire terminal-height worth of lines. 

I'd like to either disable this, or configure the number of lines shown past EOF. If it's not possible to disable, I would highly appreciate if someone could point me to the PR that introduced this change.

**Comment by pascalkuthe** [comment, marked as answer]
I filed an issue this is an unintentional regression https://github.com/helix-editor/helix/issues/6966
